### PR TITLE
link to other tools that use semgrep

### PR DESCRIPTION
**Pull request by ievans**
Documenting some of the external uses of semgrep, tagging maintainers in case they have an preferences on whether/where we link to:

- https://github.com/DefectDojo/django-DefectDojo/pull/2781 (@valentijnscholten)
- NodeJSScan / LibSast (@ajinabraham)
- SALUS (@ghbren)
- Dracon (@VJftw)

### CODE REVIEW

**Review by pabloest — APPROVED** [review]

File: docs/integrations.md [hunk 1]
```diff
@@ -229,6 +229,17 @@ docker run -v $(pwd):/src --workdir /src returntocorp/semgrep-agent:v1 python -m
     which can be changed to any value
     that `semgrep` itself understands.
 
+
+# Semgrep as an engine
+
+Many other tools use Semgrep inside, but are maintained by others outside r2c!
+
+* [NodeJSScan](https://github.com/ajinabraham/nodejsscan)
```
Comment (ajinabraham) [inline docs/integrations.md:237]: I would like to correct the usage as `nodejsscan`

**Comment by VJftw** [comment]
lgtm, thanks for this!

File: docs/integrations.md (hunk 1 above)
Comment (ievans) [reply docs/integrations.md:237]:
```suggestion
* [nodejsscan](https://github.com/ajinabraham/nodejsscan)
```

Comment (ievans) [reply docs/integrations.md]: Thanks @ajinabraham -- fixed!

File: docs/integrations.md [hunk 2]
```diff
@@ -229,6 +229,17 @@ docker run -v $(pwd):/src --workdir /src returntocorp/semgrep-agent:v1 python -m
     which can be changed to any value
     that `semgrep` itself understands.
 
+
+# Semgrep as an engine
+
+Many other tools use Semgrep inside, but are maintained by others outside r2c!
```
Comment (underyx) [inline docs/integrations.md:235]:
```suggestion
Many other tools have functionality powered by Semgrep.
Add yours [with a pull request](https://github.com/returntocorp/semgrep-docs)!
```

**Comment by nbublikov** [comment]
Hey!) import semgrep scans in defect dojo work?

**Comment by underyx** [comment]
Heya! We'll have to let the folks at DefectDojo answer that via https://github.com/DefectDojo/django-DefectDojo/pull/2781 😇

**Pull request merged**